src/Services/UserService.php

```diff
@@ -159,4 +160,31 @@ public function delete($userId)
     {
         $this->client()->delete('users/{userId}', ['userId' => $userId]);
     }
+
+    /**
+     * @param string $userId
+     * @param array|JsonSerializable|UpdatePassword $data
+     *
+     * @return User
+     */
+    public function updatePassword($userId, $data)
+    {
+        return $this->client()->post(
+            $data,
+            'users/{userId}/password',
+            [
+                'userId' => $userId,
+            ]
+        );
+    }
+
+    /**
+     * Gets currencies available to be used as a reporting currency
+     *
+     * @return array - [<currency 3 letter code> => <currency 3 letter code - currency name> ...]
```

Comment:
I actually agree with what @slavcodev is saying in principle, but shouldn't it just return the iso-4217 alpha code (3 letter code)?  It seems the API is wrong -- we can change this because the only users are internal.